db/data_migrate/20180807095900_import_customers.rb

```diff
@@ -0,0 +1,20 @@
+require 'csv'
+require 'progress_bar'
+
+puts 'Reading CSV file...'
+csv = CSV.table(Rails.root.join('db', 'data_migrate', '20180807095900_customers.csv'))
+
+puts 'Importing customers...'
+bar = ProgressBar.new(csv.count)
```

Comment:
Does this actually render properly inside the docker containers slightly-broken terminal?

Reply:
tested locally and it appears it does.